libpod/runtime.go

```diff
@@ -123,11 +124,11 @@ type RuntimeConfig struct {
 	CgroupManager string `toml:"cgroup_manager"`
 	// StaticDir is the path to a persistent directory to store container
 	// files
-	StaticDir string `toml:"static_dir"`
+	StaticDir string `toml:"-"`
 	// TmpDir is the path to a temporary directory to store per-boot
 	// container files
 	// Must be stored in a tmpfs
-	TmpDir string `toml:"tmp_dir"`
+	TmpDir string `toml:"-"`
```

Comment:
We definitely still want this - it's not tied to c/storage dirs at all

Reply:
Well it is, since you trigger events on first run after reboot,  So after reboot, we should trigger on each drivers first start.

Reply:
I don't think this was addressed - I still want to give people the ability to explicitly set StaticDir and TmpDir